assemblyline/common/tag_whitelist.yml

```diff
@@ -39,7 +51,15 @@ regex:
   network.dynamic.uri:
     - (?:ftp|http)s?://localhost(?:$|/.*)
     - (?:ftp|http)s?://(?:(?:(?:10|127)(?:\.(?:[2](?:[0-5][0-5]|[01234][6-9])|[1][0-9][0-9]|[1-9][0-9]|[0-9])){3})|(?:172\.(?:1[6-9]|2[0-9]|3[0-1])(?:\.(?:2[0-4][0-9]|25[0-5]|[1][0-9][0-9]|[1-9][0-9]|[0-9])){2}|(?:192\.168(?:\.(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])){2})))(?:$|/.*)
+    - (?:http)?:\/\/schemas\.(microsoft|openxmlformats)\.(com|org)(?:$|\/.*)
```

Comment:
I think you should split theses in two separate regex. otherwise this could lead to undesirable effect..

e.g schema.microsoft.org would be whitelisted, but we only want to whitelist: schema.microsoft.com
Same comment for static.uri section.

Reply:
Agreed, one url per line unless they target exactly the right domain

Reply:
Addressed in 10bc286fa6d819aaef9c190e57a5cecf71294139